tools/expotools/src/Versions.ts

```diff
@@ -0,0 +1,74 @@
+import { Config, Versions } from '@expo/xdl';
+
+export enum VersionsApiHost {
+  PRODUCTION = 'exp.host',
+  STAGING = 'staging.exp.host',
+}
+
+export type VersionsSchema = {
+  sdkVersions: Record<string, VersionsSdkSchema>;
+  turtleSdkVersions: {
+    android: string;
+    ios: string;
+  };
+};
+
+export type VersionsSdkSchema = Partial<{
+  androidClientUrl: string;
+  androidClientVersion: string;
+  androidExpoViewUrl: string;
+  expokitNpmPackage: string;
+  expoReactNativeTag: string;
+  facebookReactNativeVersion: string;
+  facebookReactVersion: string;
+  iosClientUrl: string;
+  iosClientVersion: string;
+  iosExpoViewUrl: string;
+  packagesToInstallWhenEjecting: Record<string, string>;
+  relatedPackages: Record<string, string>;
+  releaseNoteUrl: string;
+}>;
+
+export async function getVersionsAsync(
+  apiHost: VersionsApiHost = VersionsApiHost.STAGING
+): Promise<VersionsSchema> {
+  return await runWithApiHost(apiHost, () => Versions.versionsAsync() as Promise<VersionsSchema>);
```

Comment:
```suggestion
  return runWithApiHost(apiHost, () => Versions.versionsAsync() as Promise<VersionsSchema>);
```
Do we need to add another cycle to the EventLoop here? 🤔